src/test/java/com/cloudbees/jenkins/plugins/sshagent/SSHAgentBuildWrapperWorkflowTest.java

```diff
@@ -0,0 +1,52 @@
+package com.cloudbees.jenkins.plugins.sshagent;
+
+import com.cloudbees.jenkins.plugins.sshcredentials.SSHUserPrivateKey;
+import com.cloudbees.jenkins.plugins.sshcredentials.impl.BasicSSHUserPrivateKey;
+import com.cloudbees.plugins.credentials.CredentialsScope;
+import com.cloudbees.plugins.credentials.SystemCredentialsProvider;
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SSHAgentBuildWrapperWorkflowTest extends SSHAgentBase {
```

Comment:
Could just become a test case (method) in `SSHAgentBuildWrapperRestartingWorkflowTest`, with just a single `story.addStep` call. That would make the code a little clearer since all the Workflow-related test code would be in one suite (class).